Sources/Wrap.swift

```diff
@@ -261,7 +261,11 @@ public extension WrapCustomizable {
 public extension WrapCustomizable {
     /// Convert a given property name (assumed to be camelCased) to snake_case
     func convertPropertyNameToSnakeCase(propertyName: String) -> String {
-        let regex = try! NSRegularExpression(pattern: "(?<=[a-z])([A-Z])|([A-Z])(?=[a-z])", options: [])
+        #if os(OSX) || os(iOS) || os(watchOS) || os(tvOS)
+            let regex = try! NSRegularExpression(pattern: "(?<=[a-z])([A-Z])|([A-Z])(?=[a-z])", options: [])
```

Comment:
It'll work for now. 😄 I'd imagine that the value type `RegularExpression` will become available on Apple platforms, at which point the conditional should be removed. My other comment is applicable here as well.